charts/registry/templates/keycloak/database-credentials.yaml

```
@@ -19,10 +19,12 @@
 # SPDX-License-Identifier: Apache-2.0
 ###############################################################
 
+{{- $fullName := include "chart.fullname" . }}
+{{- $keycloak_sec_name := printf "%s-keycloak-sec" $fullName }}
 apiVersion: v1
 kind: Secret
 metadata:
-  name: keycloak-database-credentials
+  name: {{ $keycloak_sec_name }}
```

why not simply like this? Of course, this could be also applied to the other resources as well ;-) 
```suggestion
apiVersion: v1
kind: Secret
metadata:
  name: {{ include "chart.fullname . }}-keycloak-sec
```

---

Thanks for the hint :) 
Files adjusted.